README.md

```diff
@@ -1 +1,50 @@
-# Hello-World-Project
\ No newline at end of file
+# Project Name : Hello-World-Project
+
+Description of the project.
+
+The purpose of this project is to set up a "Hello world" repository. The goal
+is to master all of the tools and best practices in software development.
+
+Built With : 
+Major languages : HTML5
+Frameworks
+Technologies used
+Live Demo (if available)
+Live Demo Link
+
+Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+👤 Author1
+
+GitHub: @KingKowa
+Twitter: @kingkowa1
+LinkedIn: https://www.linkedin.com/in/frederick-ankomah-424548196
+👤 Author2
+
+GitHub: 
+Twitter: 
+LinkedIn: 
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] **Required:** Excellent Readme content, however, the formatting/styling isn't appropriate. [See this](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) and [this](https://github.com/microverseinc/readme-template) for better styling guide.